### PR TITLE
AGDIGGER-206 docs for digger node client

### PR DESCRIPTION
**Pull request by aliok**
2 commits:
1. Add some docs
2. Proper HTML generation

cc @cfoskin @PhilipGough  @laurafitzgerald

### CODE REVIEW

File: lib/api/createJob.js [hunk 1]
```diff
@@ -5,28 +5,44 @@ const authHelper = require("../util/auth");
...
+            callback(err, data);
+        });
+    } else {
+        callback("Standard template is missing!");
+    }
+
 
 };
 
+/**
```
Comment (laurafitzgerald) [inline lib/api/createJob.js:41]: Is this section referencing a method?

**Comment by laurafitzgerald** [comment]
@aliok one question in-line. Changes look fine here. I tested the `npm run docs` and this works great.

File: lib/api/createJob.js (hunk 1 above)
Comment (aliok) [reply lib/api/createJob.js:41]: @laurafitzgerald 
That's the type definition of the callback for the method above (`module.exports = function(bla)`)

In JSDoc, you use `typedef`s in the middle of no where like that as `typedef` statements are to define a type that cannot be inferred.
see http://usejsdoc.org/tags-typedef.html

Comment (laurafitzgerald) [reply lib/api/createJob.js:41]: @aliok thanks for the explanation

**Review by laurafitzgerald — APPROVED** [review]
LGTM

**Pull request merged**